maven/plugins/com.liferay.ide.maven.core/META-INF/MANIFEST.MF

```diff
@@ -6,28 +6,28 @@ Bundle-Version: 3.1.2.qualifier
 Bundle-Activator: com.liferay.ide.maven.core.LiferayMavenCore
 Bundle-Vendor: %bundle.vendor
 Require-Bundle: org.eclipse.core.runtime,
- com.liferay.ide.core;bundle-version="2.0.0";visibility:=reexport,
- org.eclipse.m2e.core;bundle-version="1.5.0";visibility:=reexport,
- org.eclipse.m2e.wtp;bundle-version="1.1.0";visibility:=reexport,
- org.eclipse.m2e.jdt;bundle-version="1.5.0",
+ com.liferay.ide.core;visibility:=reexport,
+ org.eclipse.m2e.core;visibility:=reexport,
```

Comment:
For these deps, lets specify up to the next breaking change versions.

Reply:
sorry , I don't get you , to which version?